### PR TITLE
Root cause for B's HTTP 500 responses

### PR DESCRIPTION
**Pull request by jailuthra**
**What does this pull request do? Explain your changes. (required)**

Improve logging around RTMP streamID handler (might help in narrowing down the cause of #1665)

**Specific updates (required)**

- Improve error handling logs in the HTTP Push process around RTMP stream ID handler

**How did you test each of these updates (required)**

**Does this pull request close any open issues?**

**Checklist:**

- [ ] README and other documentation updated
- [ ] Node runs in OSX and devenv
- [ ] All tests in `./test.sh` pass

### CODE REVIEW

**Review by yondonfu — APPROVED** [review]

File: server/mediaserver.go [hunk 1]
```diff
@@ -232,6 +232,7 @@ func createRTMPStreamIDHandler(s *LivepeerServer) func(url *url.URL) (strmID str
 
 			parsedProfiles, err := jsonProfileToVideoProfile(resp)
 			if err != nil {
+				glog.Error("Failed to parse JSON video profile ", err)
```
Comment (darkdarkdragon) [inline server/mediaserver.go]:
```suggestion
				glog.Errorf("Failed to parse JSON video profile err=v%", err)
```

**Review by darkdarkdragon — CHANGES_REQUESTED** [review]

**Comment by jailuthra** [comment]
Even though the HTTP500 error I was seeing in staging was most probably caused due to "Manifest already exists" as @darkdarkdragon mentioned here https://github.com/livepeer/go-livepeer/issues/1665#issuecomment-733234932

Still keeping this PR and increasing scope by improving other logs around RTMP stream ID creation as well. Many error logs in that method did not have any parameters like streamID or manifestID thus were hard to find in papertrail.

File: server/mediaserver.go [hunk 2]
```diff
@@ -285,11 +286,11 @@ func createRTMPStreamIDHandler(s *LivepeerServer) func(url *url.URL) (strmID str
 		s.connectionLock.RLock()
 		defer s.connectionLock.RUnlock()
 		if core.MaxSessions > 0 && len(s.rtmpConnections) >= core.MaxSessions {
-			glog.Error("Too many connections")
+			glog.Errorf("Too many connections for streamID url=%s err=%v", url.String(), err)
 			return nil
 		}
 		if _, exists := s.rtmpConnections[mid]; exists {
-			glog.Error("Manifest already exists ", mid)
+			glog.Errorf("Manifest %v already exists for streamID url=%s", mid, url.String())
```
Comment (darkdarkdragon) [inline server/mediaserver.go]:
```suggestion
			glog.Errorf("Manifest manifestID=%v already exists for streamID url=%s", mid, url.String())
```

**Review by darkdarkdragon — CHANGES_REQUESTED** [review]

**Review by darkdarkdragon — APPROVED** [review]
LGTM

**Pull request merged**